ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/BaseCommand.java

```diff
@@ -38,7 +38,7 @@
   protected String configFile;
 
   @Option(
-      name = {"-n", "--dry-run"},
```

Comment:
Allowing `-n` for this flag is extremely confusing since `-n` is also used for `--next` as an option in the `apply` command.